python/src/opendp/context.py

```diff
@@ -90,6 +92,40 @@
         constructors[name[5:]] = constructor, is_partial
 
 
+def register(
+    constructor: Callable[..., Union[Transformation, Measurement]],
+    name: Optional[str] = None,
+):
+    """Register a constructor function to be used in the Context API.
+
+    If the constructor supports partial application (first two arguments are ``input_domain`` and ``input_metric``),
+    then the input domain and input metric are omitted when called via the Context API.
+
+    Constructor requirements:
+    - The constructor must return a transformation or measurement.
+    - If name is None, the constructor's name must start with ``make_``.
```

Comment:
```suggestion
    * The constructor must return a transformation or measurement.
    * If name is None, the constructor's name must start with ``make_``.
```
RST doesn't convert `-` to bullet list.